### PR TITLE
Refactor fetch endpoint

### PR DESCRIPTION
**Pull request by Bartek532**
#16

### CODE REVIEW

**Comment by elte0** [comment]
Hi @Bartek532, thanks for your work! Unfortunately, you used the main branch as a starting point. I should've mentioned that there's a [develop](https://github.com/leandertolksdorf/remarkable-rss/tree/develop) branch that is ahead of main by a few commits. Sorry for that! 

Pulling into the main branch would publish immediately to the live site.

If you still want to continue working on it, I will add my comments to the code now :)

File: util/saveItemToRemarkable.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  remarkableClient,
+  folderId,
+}: {
+  item: any;
+  remarkableClient: Remarkable;
+  folderId: string;
+}) => {
+  const response = await fetch(item.link);
+  const rawHtml = await response.text();
+  const $ = cheerio.load(rawHtml);
```
Comment (elte0) [inline util/saveItemToRemarkable.ts:39]: not sure if the html cleaning part should also be an own function, as it's technically not implied by this functions name "saveItemToRemarkable".

File: util/saveItemToRemarkable.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+const pdfOptions = {
+  border: "1cm",
+};
+
+export const saveItemToRemarkable = async ({
+  item,
+  remarkableClient,
+  folderId,
+}: {
+  item: any;
```
Comment (elte0) [inline util/saveItemToRemarkable.ts:33]: I think `rss-parser` exports an `Item` type that you could use here instead of `any`.

**Comment by Bartek532** [comment]
Thanks! Ok, I will look on your comments and commit to develop branch :)

**Pull request closed without merging**